app/services/feature_flag.rb

```diff
@@ -15,8 +15,11 @@ def feature
   PERMANENT_SETTINGS = [].freeze
 
   TEMPORARY_FEATURE_FLAGS = [
-    [:service_open, 'Allow users to access the service and submit TRN Requests', 'Theodor Vararu'],
+    [:dqt_api_always_timeout, 'Always time-out the DQT API', 'Felix Clack'],
+    [:dqt_api_too_many_results, 'Always return a too many results error from the DQT API', 'Felix Clack'],
```

Comment:
I think we don't need this feature flag in the end, we only really need one if we want to test the API going down. Should we remove it?